Qausim/PropertyPro-lite
# #167279056 Refactor Request and Response Fields Cases

#### What does this PR do?
Reset request and response fields case to snakecase

#### Description of Task to be completed?
Reset field cases in tests and interfacing functions/methods of the app

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm test`
- Or run `npm run server`, then using Postman send requests to the app's endpoints in snakecase. Response should be in snakecase too.

#### Any background context you want to provide?
Requests should be sent in snakecase and response received in it

#### What are the relevant pivotal tracker stories?
[#167279056](https://www.pivotaltracker.com/story/show/167279056)

#### Screenshots (if appropriate)
Not applicable

#### Questions:
None
